frontend/src/app/shared/components/grids/grid/grid.component.html

```diff
@@ -54,21 +46,32 @@
        [ngClass] = "{'-drop-target': drag.currentlyDragging,
                      '-drop-only': drag.isDropOnlyArea(area),
                      '-resize-target': resize.isTarget(area),
-                     '-addable': add.isAddable(area),
+                     '-addable': true,
```

Comment:
@bsatarnejad is this change intentional?

Reply:
Hi @myabc 
Yes it is intended according to the AC:

> Style the "add widgets" placeholder so that it's a click zone.
> 
> Ideally, always visible in blank spaces (not only on hover)